modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -0,0 +1,262 @@
+
+== Test Runtime Traffic with Basic Authorization
+
+Run the following curl command to send traffic with the ``--user` flag for basic authorization.
+
+`curl ­-k --­­resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/tokenization -­X POST ­-H "Content­type: application/json" ­­data '[{"data": "683­31­8102", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 200 OK [{"data":"597­74­8102","status":"success"}]``
+
+You can take the tokenized SSN from above and send it back into the service. You will receive the original SSN returned. Remember that the token returned is always format preserving.
+
+`curl ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST ­-H "Content­type: application/json" ­­data '[{"data": "597­74­8102", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 200 OK [{"data":"683­31­8102","status":"success"}][root@openstackvm32 pentest­ca]``
+
+The following is an example of bad tokenization:
+
+`curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/tokenization -­X POST ­-H "Content­type: application/json" ­­data '[{"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 422 Unprocessable Entity
+[{"data":"","status":"failure","errorcode":1384,"error":"The social security number is invalid. It contains [26] characters. A social security number must have the format [###-##-####] where [\"#\"] represents a decimal digit."}]``
+
+The following is an example of bad detokenization:
+
+`curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST ­-H "Content­type: application/json" ­­data '[{"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 422 Unprocessable Entity
+[{"data":"","status":"failure","errorcode":1380,"error":"The social security number is invalid. It contains [26] characters. A social security number must have the format [###-##-####] where [\"#\"] represents a decimal digit."}]``
+
+The following is an example of bad tokenization JSON data stopped by Api Gateway protection:
+
+`curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 \https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST -­H "Content­type: application/json" ­­data '[{{{}{{{}]]"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
+
+HTTP/1.1 400 Bad Request
+{ "errorcode": 1140, "message": "Error while parsing json [line 1 char 3, byte-offset 2]: Expected member name"}``
+
+== Logs
+
+You can use the tokenization summary message that is returned in the logs to determine traffic counts. To retrieve the logs, follow the instructions in the xref:configure-log-forwarding.adoc[Runtime Fabric documentation] for configuring log forwarding.
```

Comment:
@valkyrie69 Here is the broken xref.

Reply:
Sorry @valkyrie69 I should have seen that!